cmd/repository_command_test.go

```diff
@@ -0,0 +1,136 @@
+package cmd
+
+import (
+	"bytes"
+	"log"
+	"os/exec"
+	"strings"
+	"testing"
+)
+
+// There is not any mocking. Do not run tests when use the this tool.
+
+func TestAddCommand_WhenRunsWithEmptyArgs_ReturnsOutput(t *testing.T) {
+	cmd := exec.Command("go", "run", "../main.go", "add",
+		"--cloneAddress", "",
+		"--team", "",
+		"--releaseTagPattern", "",
+		"--fixCommitPatterns", "")
+
+	out, err := cmd.Output()
+	if err != nil {
+		log.Fatal(err)
```

Comment:
```suggestion
		t.Error(err)
```

Reply:
I've made for changes you requested.

I did not test for other platforms like windows and linux but not required. Golang creates executable file for cpu architectures. Also, when we run the all tests, these for code coverage not output :)